doc/faq.mkd

```diff
@@ -50,3 +50,16 @@ on your remote repositories:
 [bitbucket-default-branch]: https://answers.atlassian.com/questions/280944/how-to-change-main-branch-in-bitbucket "How to change MAIN branch in BitBucket?"
 [gitolite-v2-default-branch]: http://stackoverflow.com/questions/7091599/git-default-remote-branch-with-gitolite "git default remote branch with gitolite"
 [gitolite-v3-default-branch]: http://stackoverflow.com/questions/13949093/git-change-default-branch-gitolite "git change default branch (gitolite)"
+
+#### I get the error: [R10K::Action::Deploy::Environment - ERROR] Couldn't update git cache for ...
```

Comment:
I believe I added it because someone in the channel had the issue, and I know I've run into it as well. I can drop it entirely if you'd like, on Monday, because it's beer 'thirty right now.